apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -200,7 +200,9 @@ type Deployment struct {
 	K8sAccessLevel K8sAccessLevel `json:"k8sAccessLevel,omitempty"`
 
 	// AutoScaler defines the configuration for the auto scaler
-	AutoScaler *AutoScaler `json:"autoScaler,omitempty"`
+	AutoScalerKeda *AutoScalerKeda `json:"autoScalerKeda,omitempty"`
```

Comment:
We can't change the name because it is already a part of the spec, we need to perhaps have both for a while, and deprecate one of them.